server/routes/contacts/manage/contact-search/contactSearchController.test.ts

```diff
@@ -256,40 +256,135 @@ describe('Contact seaarch results', () => {
     expect($('table .govuk-table__cell:eq(2)').text().trim()).toContain('England')
   })
 
-  it('should display pagination when totalPages greater than 1', async () => {
-    // Given
-    existingJourney = {
-      ...existingJourney,
-      searchContact: {
-        contact: { lastName: 'last', middleNames: '', firstName: '' },
-        dateOfBirth: { day: undefined, month: undefined, year: undefined },
-      },
-    }
-    auditService.logPageView.mockResolvedValue(null)
-    prisonerSearchService.getByPrisonerNumber.mockResolvedValue(TestData.prisoner())
-
-    const contactsArray = []
-    for (let i = 0; i < 15; i += 1) {
-      contactsArray.push(TestData.contactSearchResultItem({ id: i }))
-    }
-
-    results = {
-      ...results,
-      content: contactsArray,
-      totalElements: 15,
-      totalPages: 2,
-    }
-    contactsService.searchContact.mockResolvedValue(results)
+  describe('Pagination', () => {
+    it('should display the pagination when total pages are more than 1', async () => {
+      // Given
+      existingJourney = {
+        ...existingJourney,
+        searchContact: {
```

Comment:
would it be possible to move this section into a private data set up method and reuse in 3 tests to remove code duplication?
````
   existingJourney = {
        ...existingJourney,
        searchContact: {
          contact: { lastName: 'last', middleNames: '', firstName: '' },
          dateOfBirth: { day: undefined, month: undefined, year: undefined },
        },
      }
      auditService.logPageView.mockResolvedValue(null)
```
      prisonerSearchService.getByPrisonerNumber.mockResolvedValue(TestData.prisoner())

      const contactsArray = []
      for (let i = 0; i < 15; i += 1) {
        contactsArray.push(TestData.contactSearchResultItem({ id: i }))
      }`

Reply:
Changes added